### ISSUE TITLE
Deprecate `return_count` in ``ddb.get_by_query``.

### ISSUE THREAD

**Issue by ngr** [sosw/sosw#319]
This was one of the very shameful things in DDBC

**Comment by ngr** [comment]
Refactor get_by_query:
- change the signature of get_by_query (already commited by Nik)
- extract constructor of query args to separate function. Keep same order of kwargs and prohibit unnamed args
- Support `'Select': 'COUNT'` in the constructor if return_count. Looks like in the query args constructor we can leave this attribute.
- deprecate `return_count` attribute from `get_by_query`
- validate that the behaviour of the `get_by_query` remains unchanged. Cover the logic of the new function with relevant tests.
- create new function `count_by_query`. Use same query constructor, just call him with return_count=True.

```
        query_args = self._construct_query_args(**kwargs, return_count=True)
        response_iterator = paginator.paginate(**query_args)
        return sum([page['Count'] for page in response_iterator])
```

**Comment by ngr** [comment]
Released in [0.7.45](https://pypi.org/project/sosw/0.7.45/)
